src/paletteMachine.ts

```diff
@@ -24,8 +23,7 @@ export const paletteMachine = paletteModel.createMachine({
             return Object.values(keybindings).some(Boolean);
           }
           function eventRoseFromEditor(e: KeyboardEvent) {
-            const editorElement = document.querySelector(EDITOR_CLASSNAME);
-            return editorElement && editorElement.contains(e.target as Node);
+            return !!(e.target as Element).closest('.monaco-editor');
```

Comment:
This fixes a small additional issue - now each monaco will be detected by this check, including the one for JSON payloads. So pressing Shift+? won't open the palette when you write in that monaco instance

Reply:
I've moved this fix (and improved it) to https://github.com/statelyai/xstate-viz/pull/208 . Gonna remove the related changes from this PR in a sec